als-server/jvm/src/test/scala/org/mulesoft/als/server/lsp4j/Lsp4jLanguageServerImplTest.scala

```diff
@@ -165,6 +165,8 @@ class Lsp4jLanguageServerImplTest extends LanguageServerBaseTest with PlatformSe
             override def initialize(): Future[Unit] = Future.unit
 
             override def platform: Platform = p
+
+            override def filesInMemory: Map[String, TextDocument] = ???
```

Comment:
Map.empty better?

Reply:
this was on purpose as I do not expect the `filesInMemory` to be accessed in this tests and rather have it throw an exception instead of using an unreal value. In case this is necessary for a test I would like to be forced to implement it (but it is unlikely that this dummy class is used in those cases)